libs/service-portal/family/src/screens/UserInfo/UserInfo.tsx

```diff
@@ -9,9 +9,11 @@ import {
 } from '@island.is/island-ui/core'
 import { ServicePortalModuleComponent } from '@island.is/service-portal/core'
 import UserInfoLine from '../../components/UserInfoLine/UserInfoLine'
-import { useNatRegGeneralLookup } from '@island.is/service-portal/graphql'
-import UserInfoSidebars from './UserInfoSidebars'
+// import { useNatRegGeneralLookup } from '@island.is/service-portal/graphql'
```

Comment:
consider cleaning up these commented out lines if they are unnecessary

Reply:
Yup sorry about the size, had some difficulties in compartmentalizing each update

I agree with the images, maybe something we should bring up in tomorrow's discipline meeting